files/common/usr/bin/get-property-from-image

```diff
@@ -80,21 +80,21 @@ UNPACK_DIR=$(mktemp -d -p "$UPDATE_DIR" -t unpack.XXXXXXX)
 [[ -d "$UNPACK_DIR" ]] || die "failed to create unpack directory '$UNPACK_DIR'"
 pushd "$UNPACK_DIR" &>/dev/null || die "'pushd $UNPACK_DIR' failed"
 
-tar -x SHA256SUMS SHA256SUMS.sig.5.3 version.info -f "$UPGRADE_IMAGE_PATH" ||
+tar -x SHA256SUMS SHA256SUMS.sig.* version.info -f "$UPGRADE_IMAGE_PATH" ||
```

Comment:
Have you tested this to work? When I try it, it fails:
```
$ sudo get-property-from-image internal-dev.upgrade.tar VERSION
tar: Pattern matching characters used in file names
tar: Use --wildcards to enable pattern matching, or --no-wildcards to suppress this warning
tar: SHA256SUMS.sig.*: Not found in archive
tar: Exiting with failure status due to previous errors
get-property-from-image: failed to extract files from upgrade image '/export/home/delphix/internal-dev.upgrade.tar'
```

Reply:
Adding the `--wildcards` option might work.

If we do that, it'll successfully extract all signatures from the image; this isn't wrong, but we don't necessarily need all of them. We only need the one that matches the version of the private key installed in `/var/lib`. I'm OK using the wildcard and extracting more than is strictly necessary though, if it makes the code more simple.

Reply:
Thanks, yes for now it makes sense to keep the code simpler, so will revert back to getting the appliance version using the zfs method and change this accordingly.